### DISCUSSION TITLE
GRN (Goods Received Note)

### DISCUSSION THREAD

**Discussion by sreeisalso** [vendurehq/vendure#959] [General]
What is a Goods Received Note?
Goods Received Note is a record of goods received from suppliers, and the record is shown as a proof that ordered products had been received. Moreover, the record is used by the buyer for comparing the number of goods ordered to the ones delivered.

GRN plays an important role among various departments for several companies. Primarily, factory/store, procurement and finance/accounting departments use the record for stock updates and the payment of goods procured.

**Comment by michaelbromley** [comment]
Hi,
Thanks for explaining a goods received note. Do you have a question about Vendure?
